Libs/MRML/Logic/Testing/Cxx/vtkMRMLApplicationLogicTest1.cxx

```diff
@@ -299,8 +299,6 @@ int AddModuleLogicTest()
   {
     // Successfully removes module logic association
     appLogic->SetModuleLogic(module_name.c_str(), nullptr);
-
-    const vtkMRMLAbstractLogic* retval = appLogic->GetModuleLogic((module_name).c_str());
```

Comment:
@RafaelPalomar when introducing the change in 0279b204697563b65dcc8f25fd5c1f682c314f12, did you intended to test something ?